fvm/env.go

```diff
@@ -412,10 +412,10 @@ func (e *hostEnv) ProgramLog(message string) error {
 }
 
 func (e *hostEnv) EmitEvent(event cadence.Event) error {
-	if e.isTraceable() {
-		sp := e.ctx.Tracer.StartSpanFromParent(e.transactionEnv.traceSpan, trace.FVMEnvEmitEvent)
-		defer sp.Finish()
-	}
+	// if e.isTraceable() {
```

Comment:
I commented out these for performance reasons, but kept them in case we have to check them

Reply:
It's OK but maybe it would be better to add a comment in the code so nobody removes it.

Reply:
I'll actually add an extra flag in the context for extensive tracing instead of commenting the code